GitUI/CommandsDialogs/BrowseDialog/DashboardControl/UserRepositoriesListController.cs

```diff
@@ -8,7 +8,7 @@ public interface IUserRepositoriesListController
         Task AssignCategoryAsync(Repository repository, string? category);
         string GetCurrentBranchName(string path);
         bool IsValidGitWorkingDir(string path);
-        (IReadOnlyList<RecentRepoInfo> recentRepositories, IReadOnlyList<RecentRepoInfo> favouriteRepositories) PreRenderRepositories(Graphics g, string filter);
+        (IReadOnlyList<RecentRepoInfo> recentRepositories, IReadOnlyList<RecentRepoInfo> favouriteRepositories) PreRenderRepositories(string filter);
```

Comment:
I hope, you **do** realise what you've done here, @pmiossec! This means, we can now write tests for this! 😉

Reply:
I had that in mind also so I took the time to add some....